packages/calypso-build/package.json

```diff
@@ -22,6 +22,9 @@
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
 	"bin": "./bin/build-package.js",
+	"browserslist": [
+		"extends @wordpress/browserslist-config"
```

Comment:
Heads up @sgomes because of multiple build targets PR https://github.com/Automattic/wp-calypso/pull/30768

Reply:
Thanks, @simison! Lots of stuff to coordinate around builds lately... Let's see who ends up merging first and adjust accordingly.

Reply:
Will this have any impact on consumers of the package? They'll have to define their own `browserslist` where they build, right?

Reply:
I think so, yeah. Otherwise [babel will fall back to](https://babeljs.io/docs/en/babel-preset-env#targets)

> if no targets are specified, `@babel/preset-env` will transform all ECMAScript 2015+ code by default.

Reply:
So this would override what ever is defined in Jetpack? https://github.com/Automattic/jetpack/blob/1abc9f0d705d4b6479fb93936774c1fd3d4873af/package.json#L13-L15

(happens to be the same right now but that's not always necessarily the case)

>if no targets are specified, @babel/preset-env will transform all ECMAScript 2015+ code by default.

Wouldn't Babel pick it from `./package.json` (or `.browserslist`, or env variable...) in consumer's end (i.e. Jetpack)?